modules/ROOT/pages/xamarin/using-metrics-sdk.adoc

```diff
@@ -0,0 +1,24 @@
+= AeroGear Services Metrics SDK
```

Comment:
@johnfriz  thought you said there isn't a metrics sdk ;)

Reply:
I guess I was wrong!

@wei-lee - Is this a Xamarin only thing? I thought metrics were a part of core?

Reply:
@johnfriz It is part of the core. I think the title here is wrong. It should be part of the core module.

Reply:
We have created similar files for all other platforms, like https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/ios/using-metrics-sdk.adoc and https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/android/using-metrics-sdk.adoc. I think they all need to be updated.